src/Microsoft.ML.AutoML/TrainerExtensions/MultiTrainerExtensions.cs

```diff
@@ -240,9 +240,6 @@ internal class ImageClassificationExtension : ITrainerExtension
             ColumnInformation columnInfo, IDataView validationSet)
         {
             var options = TrainerExtensionUtil.CreateOptions<Options>(null, columnInfo.LabelColumnName);
-            options.ValidationSet = validationSet;
-            var logger = ((IChannelProvider)mlContext).Start(nameof(ImageClassificationExtension));
-            options.MetricsCallback = (ImageClassificationMetrics metric) => { logger.Trace(metric.ToString()); } ;
```

Comment:
In the future, we should work out how to get status updates all the way to Model Builder and the CLI.

Reply:
You are already getting the status updates. Image Classification API by default plumbs status updates to MLCONTEXT.LOG. The caller needs to read from there.,